### PR TITLE
Java移植组件开发博客

### PR DESCRIPTION
**Pull request by LuSingTea**
添加java-wechaty开发计划博客

Fixes # .

Make sure these boxes are checked before creating your pull request -- thank you!

- [x] Add Blog Header, including title, author, date.
e.g.

``` md
---
 title: "'Score Your Face Photo' a ML&Wechaty practice"
 date: 2017-09-18 09:00 +0800
 author: huyingxi
 ---
 > Author: [@huyingxi](https://github.com/huyingxi/wechaty_selfie) enjoying ML&Wechaty at BUPT
```

- [x] Keep all filenames & url as lowercase, and use `-` to connect words instead of space. e.g. `2017-10-06-wechat-pc-impactor` instead of `2017-10-06-WeChat PC Impactor`,
- [x] Embed the photo & video before publishing, save all external file to the blog `/download/2018` directory.
- [x] Add `<!--more-->` section to just show the abstract in blog homepage.
- [x] Add teaser image for the blog
- [x] Add your self as an author to the author data file at <https://github.com/wechaty/wechaty.github.io/blob/master/_data/authors.yml> with your GitHub username. And please also remember to reference yourself with the Github username in your blog post, so that we can have nice author information besides your blog after we published it!

[See more](https://github.com/wechaty/wechaty.github.io#contribute-guideline)

### CODE REVIEW

**Comment by lijiarui** [comment]
Please follow the blog rule:


Add Blog Header, including title, author, date.
e.g.

``` md
---
 title: "'Score Your Face Photo' a ML&Wechaty practice"
 date: 2017-09-18 09:00 +0800
 author: huyingxi
 ---
 > Author: [@huyingxi](https://github.com/huyingxi/wechaty_selfie) enjoying ML&Wechaty at BUPT
```

**Comment by lijiarui** [comment]
ping @cunkoulaocai

File: _posts/2020-07-27-java-wechaty-transplant.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+| title                     | author       | categories | image                                                        | tags         |
```
Comment (lijiarui) [inline _posts/2020-07-27-java-wechaty-transplant.md:1]: Please add blog as the template show.

**Review by lijiarui — CHANGES_REQUESTED** [review]
Please change as I request.

**Comment by lijiarui** [comment]
@cunkoulaocai I will close this PR if this PR has no reply at the end of September.

**Comment by LuSingTea** [comment]
pong

**Comment by lijiarui** [comment]
Could you change as I request as soon as possible? Since other soc contributors already posted their blog for a few weeks but you haven't posted even one blog!

I will close the PR if this PR has no new commit at the end of September.

**Comment by LuSingTea** [comment]
I encounter some format problem,I will pull request again and close this pull request

**Pull request closed without merging**